lib/public/components/Table/headers.js

```diff
@@ -18,15 +18,15 @@ import { iconCaretTop, iconCaretBottom, iconX } from '/js/src/icons.js';
  * Renders the header row
  * @param {Object} keys The full collection of API keys and their corresponding header values
  * @param {Object} model The global model object
- * @return {vnode} An array of rows containing all given header values with specific cell sizes
+ * @return {Vnode} An array of rows containing all given header values with specific cell sizes
```

Comment:
Hi @martinboulais ,

Please do not modify the documented return statements with regards to `vnode` to `Vnode` as this will split the current convention and create confusion for students joining on short term.

I would advise we keep the current form of `vnode`